account_fiscal_position_rule_sale/models/sale_order.py

```diff
@@ -0,0 +1,71 @@
+# -*- encoding: utf-8 -*-
+###############################################################################
+#
+#   account_fiscal_position_rule_sale for OpenERP
+#   Copyright (C) 2009-TODAY Akretion <http://www.akretion.com>
```

Comment:
`TODAY` is an abstract date that doesn't fit correctly I think. Better to put a fixed date.